docs/whitepapers/liquidity/matching.md

```diff
@@ -0,0 +1,96 @@
+# Matching engine
+
+This document describes the matching algorithm of the liquidity module.
+Basically, matching is done in batch-style in the liquidity module.
+Which means the timing of the order is not important like in other exchanges.
```

Comment:
The following modification is suggested. (by removing `.` and changing `W` to `w`)
```
Basically, matching is done in batch-style in the liquidity module,
which means the timing of the order is not important like in other exchanges.
```